go.mod

```
@@ -1,6 +1,6 @@
 module github.com/mongodb/mongodb-atlas-cli/atlascli
 
-go 1.24.2
+go 1.24.3
```

```suggestion
go 1.24.2
```

do you mind keeping it? our internal CI system does not support v1.24.3 yet.

---

as soon as you change it, Ill rerun the tests

---

also, we've recently updated unit tests on master, I'd apreciate if you could update your branch